index.html

```diff
@@ -0,0 +1,195 @@
+integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w==" 
+rel="stylesheet" 
+crossorigin="anonymous"
+referrerpolicy="no-referrer" />
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+<!-- Main CSS -->
+<link rel="stylesheet" href="./css/main.css" />
+
+<!-- Mobile CSS -->
+<link rel="stylesheet" href="./css/mobile.css" />
+
+<!-- Desktop CSS -->
+<link rel="stylesheet" media="(min-width:768px)" href="./css/desktop.css" />
+</head>
+
+<body class="flex-column-center">
+    <div class="upside">
+        <ul class="upside-ul">
+          <li><i class="fab fa-facebook-f white upside-item"></i></li>
+          <li><i class="fab fa-twitter white upside-item"></i></li>
+          <li><a href="" class="logout white upside-item">Logout</a></li> 
+          <li><a href="" class="mypage white upside-item">My Page</a></li>
+          <li><a href="" class="korean white upside-item">Korean</a></li>
+        </ul>
+    </div>
+    <header class="bg-light-grey flex-column-center">
+        <nav>
+            <div class="nav-background"></div>
+            <div class="light-black hamburger-menu-display" onclick="openNav()">
+                <i class="fas fa-bars hamburger-menu"></i>
+            </div>
+            <div id="myNav" class="overlay bg-light-dark">
+                    <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+                    <div class="overlay-content">
+                        <a href="./about.html">About</a>
+                        <a href="#main-program" onclick="closeNav()">Main Program</a>
+                        <a href="#speakers" onclick="closeNav()">Speakers</a>
+                        <a href="#partners-desktop" onclick="closeNav()">Partners</a>
+                        <a href="#contact" onclick="closeNav()">Contact</a>
+                    </div>
+                </div>
+                <div class="desktop-navbar">
+                    <a href="./index.html" class="main-logo">
+                        <img src="./images/logo/laxact-streaming-conference-2023.png" 
+                        alt="main logo"
+                        class="main-logo-img"/></a>
+                        <ul class="navbar-ul">
+                            <li><a href="./about.html" class="desktop-nav-link">About</a></li>
+                            <li><a href="#main-program" class="desktop-nav-link">Main Program</a></li>
+                            <li><a href="#speakers" class="desktop-nav-link">Speakers</a></li>
+                            <li><a href="#partners bg-light-black flex-column-center" class="desktop-nav-link">Partners</a></li>
+                            <li><a href="#contact" class="desktop-nav-link">Contact</a></li>
+                            <li><a href="#CCcampaign" class="CC-campaign">CC Campaign</a></li>
+                        </ul>
+                    </div>
```

Comment:
- [ ] Please add padding around the cards and decrease gap between the cards to make your cards adjust in the center instead of making them spread to full screen length.

Your design|Cindy'd design
---|---
<img width="1469" alt="Screenshot 2023-06-30 at 11 00 24 PM" src="https://github.com/katarighe/microverse-capstone-project-1/assets/44798044/9daf0b4e-bc2f-4d40-96cc-cb8457de8bcf">|<img width="1031" alt="Screenshot 2023-06-30 at 11 02 44 PM" src="https://github.com/katarighe/microverse-capstone-project-1/assets/44798044/b3248eb2-8823-48cb-9f1e-85ecf11a3b29">